examples/face_classification.py

```diff
@@ -0,0 +1,161 @@
+import random
+
+from think import Agent, Data, Environment, Memory, Motor, Task, Vision, World, Face, Result
+
+random.seed(20)
+
+
+class Condition:  # TODO where should this go
```

Comment:
i was trying not to create unnecessary new files i since he already has so many. also this is only common to my classification so common.py wouldn't make sense in the grand scheme of things.

Reply:
I guess I meant like a common.py just for the examples folder which would be private to that module